tests/slack_bolt/oauth/test_internals.py

```diff
@@ -0,0 +1,31 @@
+from slack_bolt.oauth.internals import build_detailed_error
+
+
+class TestOAuthInternals:
+    def test_build_detailed_error_invalid_browser(self):
+        result = build_detailed_error("invalid_browser")
+        assert result.startswith("invalid_browser: Perhaps, you reloaded this page or")
```

Comment:
If the above suggested language is folded in for the `invalid_browser` case, this would also need to be adjusted.

```suggestion
        assert result.startswith("invalid_browser: This can occur due to page reload, ")
```